src/Util/Crypto.php

```diff
@@ -18,8 +18,8 @@ class Crypto
 	// supported algorithms are 'sha256', 'sha1'
 	/**
 	 * @param string $data data
-	 * @param string $key  key
-	 * @param string $alg  algorithm
+	 * @param string $key key
```

Comment:
Those white spaces are actually wanted, so all the parameter comments are aligned.